scalafmt-core/shared/src/test/scala/org/scalafmt/ScalafmtRunnerTest.scala

```diff
@@ -0,0 +1,25 @@
+package org.scalafmt
+
+import scala.meta._
+import org.scalafmt.config._
+import org.scalatest.FunSuite
+
+class ScalafmtRunnerTest extends FunSuite {
+  test("sbt dialect supports trailing commas") {
+    val code = ScalafmtRunner.sbt.forSbt
+      .dialect(
+        """
+          |lazy
+          |val x = project(
+          |  a,
+          |
+          |
+          |  b,
+          |)
+        """.stripMargin
+      )
+      .parse[Source]
+      .get
+    assert(code != null)
```

Comment:
Remove, parse will never return null

Reply:
I thought I needed this to make the types align, but guess with scalatest.FunSuite you don't. Done.